components/molecule/field/src/index.js

```diff
@@ -12,16 +12,41 @@ const BASE_CLASS = 'sui-MoleculeField'
 const CLASS_INLINE = `${BASE_CLASS}--inline`
 const CLASS_AUTOHIDE = `${BASE_CLASS}--autohide`
 const CLASS_INLINE_REVERSE = `${CLASS_INLINE}-reverse`
+const CLASS_NODE_LABEL_CONTAINER = `${BASE_CLASS}-nodeLabelContainer`
 const CLASS_INPUT_CONTAINER = `${BASE_CLASS}-inputContainer`
 const CLASS_LABEL_CONTAINER = `${BASE_CLASS}-labelContainer`
 
+const MoleculeLabel = ({
+  label,
+  nodeLabel,
+  typeValidationLabel,
+  name,
+  onClickLabel
+}) => {
+  let response = null
```

Comment:
You can use a short-circuit evaluation here so there's no need to mutate `response`:

```javascript
  const MoleculeLabel = ({
    label,
    nodeLabel,
    typeValidationLabel,
    name,
    onClickLabel
  }) =>
    label &&
    ((
      <AtomLabel
        type={typeValidationLabel}
        name={name}
        text={label}
        onClick={onClickLabel}
      />
    ) ||
      (nodeLabel && (
        <div className={CLASS_NODE_LABEL_CONTAINER}>{nodeLabel}</div>
      )))
```

Reply:
I made it as is dues to readability reasons